scripts/dbUpdates.js

```diff
@@ -0,0 +1,110 @@
+//@flow
+import { sha3 } from 'web3-utils'
+import { get, delay } from 'lodash'
+import logger from '../src/imports/logger'
+import { type UserRecord } from '../src/imports/types'
+import { GunDBPublic } from '../src/server/gun/gun-middleware'
+import conf from '../src/server/server.config'
+if (process.env.NODE_ENV === 'test') process.exit(0)
+
+const UserPrivateModel = require('../src/server/db/mongo/models/user-private.js').default
```

Comment:
are there some issues which are requiring the following syntax ?
generally, line 8 `if()` could be moved just before instantiating DBUpdates and here we could just use `import`

Reply:
if we import mongodb then it tries to connect, it causes travis testing to fail

Reply:
so probably better is to move this script from postinstall and run in a separate command e.g. npm run dbupdates ?
and add `npm run dbupdates` to the deployment scripts but exclude from travis ?